### PR TITLE
Some bugs in PrintForm handle

### PR DESCRIPTION
**Pull request by LinuxTvT**
When I try to dev a sample for my printer. I see some bugs of framework about field check for indexed field. Some fields don't print because not add assignment for it.

### CODE REVIEW

File: Framework/ServiceClasses/PrinterServiceProvider/PrinterServiceClass.cs [hunk 1]
```diff
@@ -157,6 +157,7 @@ public bool LoadSingleDefinition(string definition, bool overwrite, out XFS4IoT.
                     }
                     else
                         Medias.Add(media.Name, media);
+                    PersistentData.Store<Dictionary<string, Media>>(typeof(Media).FullName, Medias);
```
Comment (seishinakano) [inline Framework/ServiceClasses/PrinterServiceProvider/PrinterServiceClass.cs:160]: Agreed, it's good to save field and media definition successfully loaded in the persistent file.

**Review by seishinakano — APPROVED** [review]
@LinuxTvT , thanks for raising PR to address issues and improvement. If you don't mind, could you tell us which company you belong to?
We will merge your changes. however, the field value assignment of changes - PrintFormHandler.cs, KAL would like to address by ourselves.

**Comment by LinuxTvT** [comment]
> 
Okie, I got it.
I work for Miraway (https://miraway.vn/). We come from Viet Nam.

**Comment by seishinakano** [comment]
> > 
> 
> Okie, I got it. I work for Miraway (https://miraway.vn/). We come from Viet Nam.

@LinuxTvT, Thank you very much for letting us know and your contribution is much appreciated.
We will merge your changes in the master branch and make further changes to prepare v1.6 package available.
Thanks again.

**Pull request merged**